File: Flutter/hksy_flutter/lib/pages/storage/api/storageapi.dart
import 'package:hksy_flutter/public/networking.dart';

class StorageApi {
  static getPerformanceDatas(userId, finish) {
    Networking.requestBase("getPerformanceDatas", finish, params: {
      "userId": userId,
    });
  }

  static getPerformanceDatasList(userId, page, limit, finish) {
    Networking.requestBase(
      "getPerformanceDatasList",
      finish,
      params: {
        "page": page,
        "limit": limit,
        "userId": userId,
      },
    );
  }

  static vipminerInfo(userId, page, limit, finish) {
    Networking.requestBase(
      "vipminerInfo",
      finish,
      params: {
        "page": page,
        "limit": limit,
        "userId": userId,
      },
    );
  }
}

File: UNI-APP/hksy_uni_3.0/pages/storage/api/storageapi.js
import networking from "../../../public/networking.js"

// 我的存储
function getPerformanceDatas(userId, completion) {
	networking.functionRequest("/getPerformanceDatas", null, {
		"userId": userId
	}, null, completion)
}

// 个人业绩(分页)
function getPerformanceDatasList(userId, page, limit, completion) {
	networking.functionRequest("/getPerformanceDatasList", null, {
		"userId": userId,
		"current": page,
		"size": limit
	}, null, completion)
}

// 超级存储基本信息(分页)
function vipminerInfo(userId, page, limit, completion) {
	networking.functionRequest("/vipminerInfo", null, {
		"userId": userId,
		"current": page,
		"size": limit
	}, null, completion)
}

module.exports = {
	getPerformanceDatas: getPerformanceDatas,
	getPerformanceDatasList: getPerformanceDatasList,
	vipminerInfo: vipminerInfo
}